spec/generators/dashboard_generator_spec.rb

```diff
@@ -61,6 +61,31 @@ class Foo < ApplicationRecord
         end
       end
 
+      it "sorts the attributes" do
+        begin
+          ActiveRecord::Schema.define do
+            create_table(:foos) do |t|
```

Comment:
How about adding a custom primary key here, providing an example for your use of `primary_key` in the exercised code?

```suggestion
            create_table(:foos, primary_key: :code) do |t|
```

Reply:
Great call - added.